README.md

```diff
@@ -20,11 +20,17 @@ We duplicated all of our data feeds across all mainnets on the Polygon testnet,
 faucet and RPC endpoints are reliable. Extending this to other testnets is not trivial, which is why we are not planning
 to do so in the foreseeable future.
 
+### Contract Addresses
```

Comment:
Two problems:
- This looks like they should enter these manually but the project imports them from packages
- It's not clear what testnet these belong to

You should have SelfServeDapiServerWhitelister addresses added to https://docs.api3.org/dapis/reference/chains.html and link the user to there.

Reply:
> It's not clear what testnet these belong to

I thought putting it under polygon-testnet section would mean its polygon-testnet but I see how it can be confusing. Addressed it in the new commit

> You should have SelfServeDapiServerWhitelister addresses added to https://docs.api3.org/dapis/reference/chains.html and link the user to there.

it looks like SelfServeDapiServerWhitelister is already in that list, I've made a reference to it in the new commit